src/ruby/devcontainer-feature.json

```diff
@@ -21,7 +21,7 @@
     "customizations": {
```

Comment:
Can you help bump the minor version? thanks!

https://github.com/devcontainers/features/blob/0deca36b58024c8edd76d17c4df4db523b1ff01a/src/ruby/devcontainer-feature.json#L3

Reply:
Done